src/main/java/com/ericsson/ei/EndpointSecurity.java

```diff
@@ -84,16 +74,20 @@ protected void configure(HttpSecurity http) throws Exception {
 
     @Override
     public void configure(AuthenticationManagerBuilder auth) throws Exception {
-        if(ldapEnabled) {
-            auth
-            .eraseCredentials(false)
-            .ldapAuthentication()
-                .userSearchFilter(ldapUserFilter)
-                .contextSource()
-                    .url(ldapUrl)
-                    .root(ldapBaseDn)
-                    .managerDn(ldapUsername)
-                    .managerPassword(decodeBase64(ldapPassword));
+        if(ldapEnabled && !ldapServerList.isEmpty()) {
```

Comment:
If we going to be correct we should check it is not null before checking isEmpty(),, if we want to avoid null pointer exception.

Or you can default ldapServerList to empty string, if property is not set:
     @Value("${ldap.server.list}") ->      @Value("${ldap.server.list:}")   ,, add ":" in the annotation after property name.

Reply:
Done